JM/CF/Scripts/3_Game/CommunityFramework/Utils/CF_Date.c

```diff
@@ -283,6 +283,15 @@ class CF_Date : Managed
 		return iTimestamp;
 	}
 
+    static void CompareDates(CF_Date date1, CF_Date date2, out int hours, out int minutes)
```

Comment:
A better name might be just Compare and then return -1 0 +1 for the default java comparator idea so you know if the dates are equal, the first is smaller or the second one is smaller.

Reply:
so you'd prefer?:
```c#
    static int Compare(CF_Date date1, CF_Date date2)
	{
		int timestamp1 = date1.GetTimestamp();
		int timestamp2 = date2.GetTimestamp();

		if (timestamp1 < timestamp2) {
			return -1;
		} else if (timestamp1 > timestamp2) {
			return 1;
		} else {
			return 0;
		}
	} 
````

Reply:
adjusted and added utc bools for compatibility with .now()